src/io/vtk/VTKMoleculeWriterImplementation.cpp

```diff
@@ -13,6 +13,7 @@
 #include <fstream>
 
 using namespace Log;
+using namespace std;
```

Comment:
even though this is a `.cpp` file where this is somewhat ok-ish I'm firmly against the practice of doing this. Reason: As soon as we allow `using namespace std;` somewhere in the code others might be tempted to use it in places where it is potentially harmful.

Reply:
FYI: https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice

Reply:
Thanks, I had googled the topic myself and found that topic. It's plausible, when you think about it